thrall/app/lib/Config.scala

```diff
@@ -43,4 +43,8 @@ object Config extends CommonPlayAppConfig {
   val healthyMessageRate = properties("sqs.message.min.frequency").toInt
 
   val dynamoTopicArn: String = properties("indexed.image.sns.topic.arn")
+
+  val auditingStreamName: String = properties("auditing.streamName")
```

Comment:
can we update the [dot properties template](https://github.com/guardian/grid/blob/master/docker/configs/generators/generators/resources/templates/dot-properties/thrall.properties.template) too?